src/libNode/ViewChangeBlockProcessing.cpp

```diff
@@ -135,15 +135,18 @@ bool Node::ProcessVCBlock(const bytes& message, unsigned int cur_offset,
     return false;
   }
 
-  if (!LOOKUP_NODE_MODE && BROADCAST_TREEBASED_CLUSTER_MODE) {
-    // Avoid using the original message for broadcasting in case it contains
-    // excess data beyond the VCBlock
-    bytes message2 = {MessageType::NODE, NodeInstructionType::VCBLOCK};
-    if (!Messenger::SetNodeVCBlock(message2, MessageOffset::BODY, vcblock)) {
-      LOG_GENERAL(WARNING, "Messenger::SetNodeVCBlock failed");
-    } else {
-      SendVCBlockToOtherShardNodes(message2);
-    }
+  // Avoid using the original message for broadcasting in case it contains
```

Comment:
I wonder if this is another leftover from pre-protobuf days (i.e., maybe now we don't have any excess data). Anyway if it's not immediately clear then we can continue recomposing the message for now.